src/keboola/facebook/api/exponential_backoff.clj

```diff
@@ -1,9 +1,9 @@
 (ns keboola.facebook.api.exponential-backoff
   (:require [keboola.docker.runtime :as runtime]))
 
-(def time-slot-ms 100)
-(def truncate 5)
-(def MAX_WAIT_TIME (* 1000 60 60 10))
+(def time-slot-ms 1000)
+(def truncate 6) ; sleep for 64 seconds at most (2^6)
+(def MAX_WAIT_TIME (* 1000 60 60 24)) ; poll for 24 hours at most
```

Comment:
este som upravil ex backoff pre pollovanie async jobov, po novom iteruje maximalne po 60 sekundach (doteraz to bolo 0.5 sekundy) a polluje async job maximalne 24 hodin (doteraz to bolo 10 hodin). Slubujem si od toho mensiu zataz na fb api v pripade ze sa dlho polluje job (x hodin). Fb api ma moze uplatnit throtling, tak sa tomu chcem vyhnut.